e2e/tests/cohorts/cohorts-review.spec.ts

```diff
@@ -19,26 +20,107 @@ describe('Cohort review tests', () => {
     await signInWithAccessToken(page);
   });
 
-  const workspace = 'e2eCohortsReviewTest';
+  const workspace = makeWorkspaceName();
+  const reviewSetNumberOfParticipants = 100;
+
+  test('Create cohort review set in cohort build page', async () => {
```

Comment:
Two tests to cover testing of create review set because create review set is accessible in two locations, cohort card snowman menu and inside the  cohort build page.